src/masonite/container/container.py

```diff
@@ -292,14 +292,15 @@ def _find_annotated_parameter(self, parameter):
         Returns:
             object -- Returns the object found in the container.
         """
+        if isinstance(parameter.annotation, str):
+            return
```

Comment:
All i did was add this. This is good yeah?

Reply:
Well it does solve the test, but then dependencies which are string type hinted and which are resolved on the fly, are going to be `None`. (I pushed a commit to demonstrate it)

It might not be an issue but it makes those changes a bit inconsistent...